__tests__/text-equals/text-equals.test.js

```diff
@@ -0,0 +1,80 @@
+import textEquals from "../../functions/equals-text/1.0";
+
+// Use following values to test
+const leftValue = "HELLO";
+const rightValue = "Hello";
+const toLower = true;
+
+// Standard comparators => Array to loop through
+const comparators = [
+  "eq",
+  "neq",
+  "cont",
+  "ncont",
+  "startsWith",
+  "endsWith",
+  "regMatch",
+];
+
+function returnExpectedResult(comparator) {
+  // Calculates the expected result for the given comparator
+  // This way the test can be automated
+  let expectedResult = false;
+  let left = leftValue;
+  let right = rightValue;
+
+  if (toLower) {
+    left = leftValue.toLowerCase();
+    right = rightValue.toLowerCase();
```

Comment:
Making the logic in the tests is not what you want. You want to test the outcome of the function that is being used

Reply:
So please refactor tests to a similar approach being used of the functions in the main branch